src/store/selectors/index.js

```diff
@@ -1,19 +1,16 @@
 import { createSelector } from 'reselect'
 import mapValues from 'object-loops/map'
 
-const currentCaptures = (state, props) => {
-  const method = props.method
-  const side = props.side ? props.side : null
-  return state.captures[method].filter(c => c.side === side)
-}
+const currentCaptures = (state, { method, side = null }) =>
+  state.captures[method].filter(c => c.side === side)
 
 const outputCaptures = (state) => {
   const captures = state.captures
```

Comment:
I considered this, but stuck with state because it is a selector and selector's always take state as their first argument.

Therefore not having state as the explicit first argument makes it less readable for me